### PR TITLE
feat: add missing python sdk guides based on DatahubGraph

### PR DESCRIPTION
**Pull request by yoonhyejin**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

resolves DR-429

### CODE REVIEW

File: docs/api/graphql/getting-started.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Note that by default Elasticsearch only allows pagination through 10,000 entitie
 If you need to paginate through more, you can change the default value for the `index.max_result_window` setting in Elasticsearch, or using the scroll API to read from the index directly.
 :::
 
-## Modifying an Entity: Mutations
+## an Entity: Mutations
```
Comment (hsheth2) [inline docs/api/graphql/getting-started.md:72]: this doesnt look right

File: docs/api/tutorials/modifying-dataset-custom-properties.md [hunk 2]
```diff
@@ -48,13 +50,14 @@ datahub get --urn "urn:li:dataset:(urn:li:dataPlatform:hive,fct_users_deleted,PR
...
 ## Add Custom Properties programmatically
+
 The following code adds custom properties `cluster_name` and `retention_time` to a dataset named `fct_users_deleted` without affecting existing properties.
 
 <Tabs>
 <TabItem value="graphql" label="GraphQL">
 
 > 🚫 Adding Custom Properties on Dataset via GraphQL is currently not supported.
-> Please check out [API feature comparison table](/docs/api/datahub-apis.md#datahub-api-comparison) for more information, 
+> Please check out [API feature comparison table](/docs/api/datahub-apis.md#datahub-api-comparison) for more information,
```
Comment (hsheth2) [inline docs/api/tutorials/modifying-dataset-custom-properties.md:60]: trailing comma

File: docs/api/tutorials/modifying-dataset-custom-properties.md
Comment (hsheth2) [inline docs/api/tutorials/modifying-dataset-custom-properties.md:1]: when I heard "modifying" I think "updates"

I was surprised that adding/removing props is also covered by this guide

**Review by shirshanka — APPROVED** [review]
LGTM.

**Pull request merged**